packages/wonder-blocks-grid/docs.md

```diff
@@ -117,27 +117,13 @@ Additionally, the following grid size specs are also available:
 | ----- | ------- | ------ | ------ | -------------------------------------------- |
 | large | 12      | 32px   | 16px   | `min-width: 1px` (No maximum content width.) |
 
-**12-Column Modal (`GRID_MODAL_12_SPEC`)**
+**12-Column Modal (`GRID_MODAL_SPEC`)**
```

Comment:
I thought we were completely abandoning the grid for modals.

Reply:
Nope - still using it! We're just using it for its viewport switching abilities, but not really any of the lower-level primitives.